Require state/nonce to only contain URL safe characters (#126)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

!---
Figure: Reference Design for Response Mode `direct_post`

> (1) The Verifier generates at least 16 fresh, cryptographically random bytes with sufficient entropy, associates it with the session and base64url encodes it for use in the nonce parameter.

Reviewer: ```suggestion
(1) The Verifier produces a `nonce` value by generating at least 16 fresh, cryptographically random bytes with sufficient entropy, associates it with the session and base64url encodes it.
```
